pkg/render/typha.go

```diff
@@ -256,6 +256,7 @@ func (c *typhaComponent) typhaRole() *rbacv1.ClusterRole {
 					"ipreservations",
 					"networkpolicies",
 					"networksets",
+					"egressgatewaypolicies",
```

Comment:
This doesn't belong here - it needs to go below in the Calico Enterprise only section.

Reply:
I cannot find EE only section in this file. At the same time, it seems `ExternalNetwork` resource which is also EE only is listed here.

Reply:
This is where it should be: https://github.com/tigera/operator/blob/cfa369163949ec3281c5240fc0bea55d45e3bee4/pkg/render/typha.go#L314-L341

ExternalNetworks is a bug and also shouldn't be here - could you move that as well?

Reply:
sure, will move it.

Reply:
Fixed.